server/models/Collective.js

```diff
@@ -1769,7 +1768,10 @@ export default function (Sequelize, DataTypes) {
       if (!newHostCollective.isHostAccount) {
         await newHostCollective.becomeHost();
       }
-      return this.addHost(newHostCollective, creatorUser);
+      await this.addHost(newHostCollective, creatorUser);
+      // Make sure the hostFeePercent is updated when changing hosts
+      await this.update({ hostFeePercent: newHostCollective.hostFeePercent });
```

Comment:
Why are we not doing that in `addHost` ?

Reply:
This is an edge-case related to the fact that we want to inherit the fee when creating a collective, not when changing its host.
The problem is that `changeHost` deletes the collective `HostCollectiveId` before calling `addHost`, making it impossible for `addHost` to detect if this collective is being created or is just changing host.

Reply:
I don't see (or don't remember) why we don't want to update the `hostFeePercent` when changing the host. It would make sense, no?

Reply:
@znarf, we do, and that's exactly what is happening. (We're actually testing for that behavior)
This change is to make that possible while also being possible to inherit the `hostFeePercent` when creating a new collective.
Otherwise, `addHost()`, which is used in the case we add a new host and when we change host, won't have the necessary context to decide if the `hostFeePercent` should be inherited from the Host or from the parent Collective.